aviatrix-controller-initialize/main.tf

```diff
@@ -22,7 +18,7 @@ resource "aws_iam_role" "iam_for_lambda" {
 EOF
 }
 
-resource "aws_iam_policy" "lambda-policy" {
+resource aws_iam_policy lambda-policy {
```

Comment:
Do all versions of terraform 0.12.x support this?

Reply:
I think so as I tested this also on 0.12.0